### DISCUSSION TITLE
Privacy question to the minimum release age feature

### DISCUSSION THREAD

**Discussion by Mitgenosse** [renovatebot/renovate#23847] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.159.7

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

I would like to get a better understanding on how the [minimumReleaseAge](https://docs.renovatebot.com/configuration-options/#minimumreleaseage) feature works. Renovate itself is working like charm within a self-hosted gitlab instance!

When a new merge request is created in a renovate-managed repository, I see something that looks like a new pipeline job.

![Screenshot 2023-08-14 at 11 34 39](https://github.com/renovatebot/renovate/assets/11715759/84fbef42-948b-48f7-9c1c-6ca302db09fb)

I know that this job will be green as soon as the release age passes the configured value, but how does it work exactly? Does it do API calls to some external third-party service? Does it expose any data to external systems out of my/our control? Or is this set by the renovate run from within our own network in which we host it?

Thank you very much for your time!

### What you tried so far.

I read the documentation but it couldn't enlighten me. I also read the [three privacy-related discussions](https://github.com/renovatebot/renovate/discussions?discussions_q=Privacy+question) which I could find.

**Comment by rarkins** [comment, marked as answer]
It does not send any data to Mend or others, it uses the release timestamp from the registry for each version

**Comment by HonkingGoose** [comment]
> ## If you're self-hosting Renovate, tell us what version of Renovate you run.
> 
> 35.159.7

We released Renovate `36.0.0` a while back. You should probably update. 😉 Read the [Renovate docs, release notes for version 36](https://docs.renovatebot.com/release-notes-for-major-versions/#version-36) to see the breaking changes.
